phpBB/language/en/acp/permissions_phpbb.php

```diff
@@ -79,6 +79,7 @@
 	'ACL_U_SAVEDRAFTS'	=> 'Can save drafts',
 	'ACL_U_CHGCENSORS'	=> 'Can disable word censors',
 	'ACL_U_SIG'			=> 'Can use signature',
+	'ACL_U_EMOJI'		=> 'Can use Emoji in topic title',
```

Comment:
Emoji => emojis

Reply:
I would still stick with just `emoji`, as it's the plural form aswell.

Reply:
According to those example sentences: https://www.lexico.com/definition/emoji the "emojis" is for plural more common.